src/wallet/rpcwallet.cpp

```diff
@@ -1285,7 +1285,8 @@ UniValue addwitnessaddress(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
     {
         std::string msg = "addwitnessaddress \"address\" ( p2sh )\n"
-            "\nAdd a witness address for a script (with pubkey or redeemscript known). Requires a new wallet backup.\n"
+            "\nDEPRECATED: use option -addresstype=[bech32|p2sh-segwit] instead.\n"
+            "Add a witness address for a script (with pubkey or redeemscript known). Requires a new wallet backup.\n"
```

Comment:
Doesn't require a new wallet backup anymore, IIRC.

Reply:
I don't think it makes sense to update the documentation of a deprecated call, besides adding a DEPRECATED message and hiding it.

Reply:
Agreed, its not like backing up would actually be a bad thing anyway